docs/source/modifying-messages.rst

```diff
@@ -0,0 +1,390 @@
+====================
+Message manipulation
+====================
+
+To create malformed messages tlsfuzzer supports modifying them before sending.
```

Comment:
Tlsfuzzer provides facilities to modify messages before sending, this is used to create malformed messages.

Reply:
rephrased it a bit more, please recheck